docs/eventing/samples/kafka/channel/README.md

```diff
@@ -9,24 +9,12 @@ You can install and configure the Apache Kafka CRD (`KafkaChannel`) as the defau
 
 ## Prerequisites
 
-You must ensure that you meet the [prerequisites listed in the Apache Kafka overview](../README.md).
-
-You must also have the following tools installed:
-- `curl`
-- `sed`
+- Ensure that you meet the [prerequisites listed in the Apache Kafka overview](../README.md).
+- A Kubernetes cluster with [Knative Kafka Channel installed](../../../../install/README.md).
```

Comment:
It looks like if you naively just install the strimzi operator it installs with replicas of 1, and I think that means that all of the examples that we have use replication of 3 and I think they will fail? Perhaps we should mention that in the install guide, or here, or ? Or maybe I just did something wrong?

Reply:
@vaikas the partitions is _not_ tied to the nr of nodes.

the replicationFactor is, though

Reply:
Ok, I just remember I tried to very different configurations and got bit, thanks!!!